### PR TITLE
LOX24 SMS Gateway Notifier

### PR DESCRIPTION
**Pull request by alebedev80**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | https://github.com/symfony/symfony-docs/pull/19692

### CODE REVIEW

File: symfony/lox24-notifier/6.4/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      "position": "after_target",
+      "warn_if_missing": true,
+      "target": "        texter_transports:",
+      "content": "            lox24: '%env(LOX24_DSN)%'"
+    }
+  ],
+  "env": {
+    "#1": "LOX24_DSN=lox24://TOKEN@default?from=FROM"
+  }
+}
```
Comment (OskarStark) [inline symfony/lox24-notifier/6.4/manifest.json:14]: Please add new line

Comment (OskarStark) [inline symfony/lox24-notifier/6.4/manifest.json:14]: Please add new line

**Review by OskarStark — COMMENTED** [review]
folder must be 7.1 not 6.4

File: symfony/lox24-notifier/6.4/manifest.json (hunk 1 above)
Comment (alebedev80) [reply symfony/lox24-notifier/6.4/manifest.json]: Done

File: symfony/lox24-notifier/7.1/manifest.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    {
+      "file": "config/packages/notifier.yaml",
+      "position": "after_target",
+      "warn_if_missing": true,
+      "target": "        texter_transports:",
+      "content": "            lox24: '%env(LOX24_DSN)%'"
+    }
+  ],
+  "env": {
+    "#1": "LOX24_DSN=lox24://TOKEN@default?from=FROM"
```
Comment (OskarStark) [inline symfony/lox24-notifier/7.1/manifest.json:12]:
```suggestion
    "#1": "LOX24_DSN=lox24://USERNAME:TOKEN@default?from=FROM"
```
?

File: symfony/lox24-notifier/7.1/manifest.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (OskarStark) [inline symfony/lox24-notifier/7.1/manifest.json:1]: indentation must be fixed

Comment (alebedev80) [reply symfony/lox24-notifier/7.1/manifest.json:1]: Fixed.

Fixed 'USERNAME' to 'USER'. In LOX24 user has integer id only, not username.

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by alebedev80** [comment]
Thank you all!

**Pull request merged**